### PR TITLE
Feature/private registry

### PR DESCRIPTION
**Pull request by varun257**
Private registry api endpoints

### CODE REVIEW

File: src/cloud/project/containerRegistry/cloud-project-containerRegistry.service.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+angular.module("ovh-api-services").service("OvhApiCloudProjectContainerRegistry", function ($injector) {
+    "use strict";
+
+    return {
+        v6: function () {
+            return $injector.get("OvhApiCloudProjContainerRegistryV6");
+        },
+        users: function () {
```
Comment (marie-j) [inline src/cloud/project/containerRegistry/cloud-project-containerRegistry.service.js]:
```suggestion
        Users: function () {
```
to be consistent

File: src/cloud/project/containerRegistry/cloud-project-containerRegistry.v6.service.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+angular.module("ovh-api-services").service("OvhApiCloudProjContainerRegistryV6", function ($cacheFactory, $resource) {
```
Comment (marie-j) [inline src/cloud/project/containerRegistry/cloud-project-containerRegistry.v6.service.js]:
```suggestion
angular.module("ovh-api-services").service("OvhApiCloudProjectContainerRegistryV6", function ($cacheFactory, $resource) {
```

**Comment by varun257** [comment]
@marie-j  have renamed. Please have a look

**Comment by ovh-cds** [comment]
CDS Report build#910.0 ✘  
* 
  * build ✔ 
  * test ✘

**Comment by ovh-cds** [comment]
CDS Report build#911.0 ✘  
* 
  * build ✔ 
  * test ✘

**Comment by antleblanc** [comment]
@varun257 Build fail due to some ESLint errors:)

File: src/cloud/project/cloud-project.service.js [hunk 3]
```diff
@@ -66,7 +66,10 @@ angular.module("ovh-api-services").service("OvhApiCloudProject", function ($inje
         },
         Quota: function () {
             return $injector.get("OvhApiCloudProjectQuota");
-        }
+        },
+        ContainerRegistry: function () {
+            return $injector.get("OvhApiCloudProjectContainerRegistry");
+        },
```
Comment (Jisay) [inline src/cloud/project/cloud-project.service.js]: ```error  Unexpected trailing comma  comma-dangle```

File: src/cloud/project/containerRegistry/cloud-project-containerRegistry.service.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+angular.module("ovh-api-services").service("OvhApiCloudProjectContainerRegistry", function ($injector) {
+    "use strict";
+
+    return {
+        v6: function () {
+            return $injector.get("OvhApiCloudProjectContainerRegistryV6");
+        },
+        Users: function () {
+            return $injector.get("OvhApiCloudProjectContainerRegistryUsers");
+        },
```
Comment (Jisay) [inline src/cloud/project/containerRegistry/cloud-project-containerRegistry.service.js]: ```error  Unexpected trailing comma  comma-dangle```

File: src/cloud/project/containerRegistry/users/cloud-project-containerRegistry-users.service.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+angular.module("ovh-api-services").service("OvhApiCloudProjectContainerRegistryUsers", function ($injector) {
+    "use strict";
+
+    return {
+        v6: function () {
+            return $injector.get("OvhApiCloudProjectContainerRegistryUsersV6");
+        },
```
Comment (Jisay) [inline src/cloud/project/containerRegistry/users/cloud-project-containerRegistry-users.service.js]: ```error  Unexpected trailing comma  comma-dangle```

**Pull request merged**